rabbitmq/src/messages.rs

```diff
@@ -79,6 +81,22 @@ impl ResultMessage {
             head_result,
             run_id,
             worker_name,
+            is_success,
+        }
+    }
+
+    pub fn aborted(run_id: Uuid, worker_name: String, error: String) -> Self {
+        Self {
+            run_id,
+            worker_name,
+            is_success: false,
+            download_result: Err(DownloadError {
+                error: error.clone(),
+            }),
+            ping_result: Err(PingError {
+                error: error.clone(),
+            }),
+            head_result: Err(HeadError { error }),
```

Comment:
This repetition here suggests that maybe we should have our types defined somewhat differently. Perhaps `ResultMessage` should have a single `result: Result<DetailedResults, Error>`, with detailed results containing current download/ping/head results?
Just a suggestion, feel free to ignore.

Reply:
that's actually good point, but requires quite a lot of changes, I will revise this approach once I get to using `is_success` and have a reference to treat errors differently, for now its a valid json in db